pom.xml

```diff
@@ -1,14 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>dev.snowdrop</groupId>
-    <artifactId>snowdrop-dependencies</artifactId>
-    <version>2.5.12.Final</version>
-  </parent>
   <groupId>dev.snowdrop.example</groupId>
   <artifactId>istio-distributed-tracing</artifactId>
-  <version>2.5.12-3-SNAPSHOT</version>
+  <version>2.11.0-SNAPSHOT</version>
```

Comment:
I think it should be 2.7.2-SNAPSHOT because the version has been always driven by the Spring Boot version, not the Dekorate version which is an utility only.

Reply:
The reason for version `2.11.0` is that we're now aligning the examples to the Dekorate version.